File: src/main/java/com/lawencon/elearn/dto/examschedule/ExamScheduleInsertResDto.java
package com.lawencon.elearn.dto.examschedule;

public class ExamScheduleInsertResDto {
	private ExamScheduleInsertDataResDto data;
	private String message;
	public ExamScheduleInsertDataResDto getData() {
		return data;
	}
	public void setData(ExamScheduleInsertDataResDto data) {
		this.data = data;
	}
	public String getMessage() {
		return message;
	}
	public void setMessage(String message) {
		this.message = message;
	}
}

File: frontend/src/app/dto/exam-schedule/exam-schedule.insert.res.dto.ts
import { ExamScheduleInsertDataResDto } from "./exam-schedule.insert.data.res.dto";

export class ExamScheduleInsertResDto{
    private _data! : ExamScheduleInsertDataResDto
	private _message! : string

    public get data(): ExamScheduleInsertDataResDto {
        return this._data;
    }

    public set data(data: ExamScheduleInsertDataResDto) {
        this._data = data;
    }

    public get message(): string {
        return this._message;
    }

    public set message(message: string) {
        this._message = message;
    }

}